### ISSUE TITLE
 Is it sensitive to batch size

### ISSUE THREAD

**Issue by burcehan** [zeke-xie/stable-weight-decay-regularization#1]
Hi ,Thanks for your work,

Is it sensitive to batch size,can I use small batch size?
Thanks

**Comment by zeke-xie** [comment]
Hi burcehan,

Stable Weight Decay is robust to batch size.
It is totally fine to use small batch size. 

When the batch size is small, our experiment (training ResNet18 on CIFAR-10) suggests that the advantage of AdamS over AdamW and Adam is even more significant.
